pandas/_libs/tslibs/nattype.pyx

```diff
@@ -435,6 +434,21 @@ class NaTType(_NaT):
     # The remaining methods have docstrings copy/pasted from the analogous
     # Timestamp methods.
 
+    strftime = _make_error_func(
+        "strftime",
+        """
+        Timestamp.strftime(format)
+
+        Return a string representing the given POSIX timestamp
```

Comment:
can you add the link to the format strings (pointed at here: https://pandas.pydata.org/docs/reference/api/pandas.DatetimeIndex.strftime.html?highlight=strftime#pandas.DatetimeIndex.strftime)

Reply:
I have added the link